src/libcore/ptr/mod.rs

```diff
@@ -668,7 +668,18 @@ pub unsafe fn read<T>(src: *const T) -> T {
 /// ```
 ///
 /// Accessing unaligned fields directly with e.g. `packed.unaligned` is safe however.
-// FIXME: Update docs based on outcome of RFC #2582 and friends.
+///
+/// # Examples
+///
```

Comment:
I don't know if this is general docs style, but my impression (and preference) is that examples come with a brief one-line summary of what the example is doing. Something like "{Read,Write} an usize value {from,to} a byte buffer", perhaps?